examples/client_ws.py

```diff
@@ -43,10 +34,18 @@ async def dispatch():
 
                 break
 
-    # send request
     async with aiohttp.ClientSession() as session:
         async with session.ws_connect(url, autoclose=False, autoping=False) as ws:
-            await dispatch()
+            # send request
+            dispatch_task = asyncio.create_task(dispatch(ws))
+
+            # Exit with Ctrl+D
+            while line := await asyncio.to_thread(sys.stdin.readline):
```

Comment:
An alterative would be:
```
            reader = asyncio.StreamReader()
            await asyncio.get_running_loop().connect_read_pipe(
                lambda: asyncio.StreamReaderProtocol(reader), sys.stdin)

            while line := await reader.readline():  # Exit with Ctrl+D
```

But, from the documentation:
> Essentially, transports and protocols should only be used in libraries and frameworks and never in high-level asyncio applications.

So, maybe `to_thread()` is the safer option...